consensus/polybft/signer/common.go

```diff
@@ -1,25 +0,0 @@
-package bls
-
-import (
-	"encoding/hex"
-)
-
-var (
-	domain, _ = hex.DecodeString("508e30424791cb9a71683381558c3da1979b6fa423b2d6db1396b1d94d7c4a78")
```

Comment:
Is it worth to keep domain there?

Reply:
That file is deleted. Everything is inside bnsnark1 repo

Reply:
Is the Domain inside bnsnark1? I thought the domain is something application specific.

Reply:
Yes, I have provided method to setup domain if needed something different for it. But I put value we currently use as a default value